ccd-definition/FixedLists/RespondentResponseType.json

```diff
@@ -20,7 +20,7 @@
   {
     "ID": "RespondentResponseType",
     "ListElementCode": "COUNTER_CLAIM",
-    "ListElement": "Reject all of the claim and wants to counterclaim",
+    "ListElement": "Defends all of the claim and wants to counterclaim",
```

Comment:
I believe this has to be removed, because it damages journey

Reply:
done